src/extensions/cli/cli.ts

```diff
@@ -1,69 +1,32 @@
-'use strict'
+import { spawn } from 'node:child_process'
+import path from 'node:path'
+import { isNotNullish } from '../../utils/index.js'
 
-/**
- * The CLI helper used by the CLI extension.
- *
- * @module extensions/Cli/Cli
- */
-
-const { spawn } = require('child_process')
-const path = require('path')
+export type CliArgs = ConstructorParameters<typeof Cli>
 
-/**
- * Cli extension.
- *
- * @class
- */
 class Cli {
+    public cwd: string
+    public env: Record<string, string>
+    public killSignal: number | NodeJS.Signals | undefined
+    public killDelay: number
+    public exitCode: number | undefined
+    public stdout: string
+    public stderr: string
+
     constructor() {
-        /**
-         * The Current Working Directory.
-         *
-         * @type {string}
-         */
         this.cwd = process.cwd()
-
-        /**
-         * An object containing environment variables to inject when running your command.
-         *
-         * @type {Object}
-         */
         this.env = {}
-
-        /** @type {string} */
-        this.killSignal = null
-
-        /** @type {number} */
+        this.killSignal = undefined
         this.killDelay = 0
-
-        /**
-         * Latest command execution exit code.
-         *
-         * @type {number}
-         */
-        this.exitCode = null
-
-        /**
-         * The command's output.
-         *
-         * @type {string}
-         */
+        this.exitCode = undefined
         this.stdout = ''
-
-        /**
-         * The command's error output.
-         *
-         * @type {string}
-         */
         this.stderr = ''
     }
 
     /**
      * Sets the Current Working Directory for the command.
-     *
-     * @param {string} cwd - The new CWD
```

Comment:
I would let jsdoc too

Reply:
Normally, we will get a warning if we use JSDoc in the TypeScript project. (It is not recommended)
Unless you want me to keep the comment (but not the @param)